influxdb/src/client/mod.rs

```diff
@@ -222,24 +223,24 @@ impl Client {
                 url.query_pairs_mut()
                     .append_pair("precision", &write_query.get_precision());
 
-                ReqwestClient::new().post(url).body(query.get())
+                isahc::post_async(query.get(), url.as_str().to_owned()).await
```

Comment:
If my IDE doesn't trick me the parameters are the wrong way around. First argument should be url and second should be body.